pom.xml

```diff
@@ -287,5 +287,25 @@
        <url>https://repo.jenkins-ci.org/public/</url>
      </pluginRepository>
   </pluginRepositories>
+
+  <profiles>
+    <profile>
+      <id>java8-doclint-disabled</id>
```

Comment:
Thanks for the clarification.  I've generally treated the 2.6.x branch (and the git client plugin 1.21.x branch) as "must be compiled with JDK 7".  There were problems in one of those two environments which forced me to stay with JDK 7.  I don't remember which, nor do I remember the details of the issues.

Many javadoc fixes were needed on the master branch.  I support this exclusion on the 2.6.x branch because it is a branch with limited changes.